notebooks/239-image-bind/README.md

```diff
@@ -10,13 +10,26 @@ In this tutorial, we consider how to convert and run ImageBind model using OpenV
 
 ## Notebook Contents
 
-The tutorial consists of following steps:
+This folder contains two notebooks that show how to optimize and quantize model with OpenVINO:
```

Comment:
I propose to align it everywhere with OV documentation where we agreed to have the convert step (MO+FEs) and the optimize step (NNCF). I.e. 1st notebook is about the conversion to IR and 2nd is about the optimization by 8-bit quantization.

Reply:
probably if we will agreed to upload model somewhere and it will be applicable (not image bind case), third simple notebook for demo with opportunity to select model (fp/int8) can take place if we speak about general structure